packages/jitm/src/JITM.php

```diff
@@ -238,6 +243,14 @@ function( $connected_admin ) {
 
 						submitBtn.disabled = true;
 
+						// Track it
```

Comment:
How about adding tracking in PHP, to the `set_connection_owner` function directly, as the disconnection happens?

Reply:
in the endpoint callback? hmm yeah that sounds like a more complete solution, good idea.

Reply:
Done

Reply:
Updated the PR description to reflect the changes. @jeherve please remember to test the Calypso UI now for switching connection owners, since it also depends on the endpoint.